styles/styles.css

```diff
@@ -0,0 +1,83 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+
+body {
+  background-color: rgb(23, 23, 23);
+}
+
+.logo {
+  padding: 1.2rem;
+}
+
+.logo:hover {
+  color: #ff6b00;
+  cursor: pointer;
+}
+
+i {
+  padding: 20px;
+}
+
+i:hover {
+  cursor: pointer;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: rgb(87, 87, 87);
+  color: white;
+}
+
+div ul {
+  display: none;
+}
+
+li {
+  list-style: none;
+}
+
+ul li a {
+  text-decoration: none;
+  padding: 10px;
+}
+
+.container2 {
+  display: flex;
+  flex-direction: column;
+  color: white;
+}
+
+.section1 h1 {
+  color: #ff6b00;
+  font-size: 56px;
+  margin-top: 60px;
+  padding: 1rem;
+}
+
+.section1 h2 {
+  font-size: 20px;
+  padding: 1rem;
+}
+
+.section1 p {
+  font-size: 16px;
+  padding: 20px 20px;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```

Comment:
- [ ] Please 🙏 stick to the design template as much as possible. Make sure you use appropriate margins and padding to make your page look identical to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=tG1jh7hS4l814OsZ-0)[requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

**_Your design_**

<img width="276" alt="Screenshot 2023-02-17 at 12 20 59 AM" src="https://user-images.githubusercontent.com/44798044/219468291-a3d079ef-cbb7-4a28-9d65-a55d907430ad.png">

_**Figma design:**_

<img width="383" alt="Screenshot 2023-02-16 at 11 31 35 PM" src="https://user-images.githubusercontent.com/44798044/219466322-4c402cc2-c084-4521-9fd6-a8cf74440db0.png">